pulp_rpm/app/models/modulemd.py

```diff
@@ -40,7 +41,7 @@ class Modulemd(Content):
         dependencies (Text):
             Module dependencies, if any.
         artifacts (Text):
-            Artifacts shipped with this module.
+        _pulp_domainfacts shipped with this module.
```

Comment:
Not sure what happened here.
```suggestion
        Artifacts shipped with this module.
```

Reply:
Prob a PEBKAC while I was fixing all the pulp_domains to _pulp_domains